parser-typechecker/src/Unison/Codebase.hs

```diff
@@ -290,15 +290,15 @@ installUcmDependencies c = do
 -- if it makes sense to later.
 addDefsToCodebase ::
```

Comment:
Is the goal for this to also be in Transaction then?
I'm  guessing we just need to also convert some more methods before we can do that?

Reply:
Yep.